pkgs/tools/misc/fortune/extensions/blag-fortune.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, stdenv, fetchgit, fortune }:
+
+stdenv.mkDerivation rec {
+  pname = "blag-fortune";
+  version = "2022-07-16";
+
+  # We use fetchurl instead of fetchFromGitHub because the release pack has some
+  # special files.
+  src = fetchgit {
+    url = "https://notabug.org/PangolinTurtle/BLAG-fortune.git";
+    rev = "b07fb0312f";
+    sha256 = "sha256-Fi8o+hd2frBXXfzjwiJC7/2ZjdEc8vJbB7bSh9n3Vx0=";
+  };
```

Comment:
This comment doesn't make sense because the upstream doesn't use GitHub. Also I think you should pull a tag and not a specific revision. If you want to pull a tag, when the package version should be `unstable-YYYY-MM-DD`.